wordpress/wp-content/plugins/memberful-wp/src/paywall/renderer.php

```
@@ -0,0 +1,368 @@
+      MEMBERFUL_URL . '/stylesheets/paywall.css',
+      array(),
+      MEMBERFUL_VERSION
+    );
+
+    if ( function_exists( 'wp_is_block_theme' ) && wp_is_block_theme() ) {
+      return;
+    }
+
+    wp_enqueue_script(
+      'memberful-paywall-banner',
+      MEMBERFUL_URL . '/js/src/paywall-banner.js',
```

PHPStan (2.1.54)

[warning] Constant MEMBERFUL_URL not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)